nilearn/decoding/decoder.py

```diff
@@ -176,7 +178,9 @@ def _default_param_grid(estimator, X, y):
     # define sensible default for different types of estimators
     if isinstance(estimator, (RidgeCV, RidgeClassifierCV)):
         param_grid["alphas"] = [np.geomspace(1e-3, 1e4, 8)]
-    elif isinstance(estimator, (LogisticRegression, LinearSVC, SVR)):
+    elif isinstance(estimator, LogisticRegressionCV):
+        param_grid["Cs"] = [np.geomspace(2e-3, 2e4, 8) * min_c]
```

Comment:
@bthirion for your suggested change, the reason I used `2e-3`/`2e4` instead of `1e-3`/`1e4` was to be consistent with [the previous default hyperparameter values](https://github.com/nilearn/nilearn/blob/main/nilearn/decoding/decoder.py#L179), which used `[2, 20, 200] * min_c`. 

From what I understand, the `[2, 20, 200]` is multiplied by the `min_c` value which is set to 0.5 unless the estimator uses L1 penalty, in which case `min_c` is computed with [`sklearn.svm.l1_min_c`](https://scikit-learn.org/stable/modules/generated/sklearn.svm.l1_min_c.html). So the default (for the L2 regression case) was in fact `[1, 10, 100]`, just obtained in a more convoluted way. For the L1 penalty case, the `min_c` is obtained in a more data-driven way, though it's unclear to me why it's multiplied by `[2, 20, 200]` instead of something like `[1, 10, 100]`.

Reply:
Maybe adding a slight comment on this in the code is useful for future reference.
Normally, the grid should be such that we should not have to care about what exact grid was used, and I don't really have an opinion on this.

Reply:
Ok, I added a comment, hopefully it makes things clearer